### DISCUSSION TITLE
immorality/cruelty/human/

### DISCUSSION THREAD

**Discussion by giscus** [SageofPHY6Paths/hinduismdebunked.com#16] [Comments]
# immorality/cruelty/human/

A website that debunks Hinduism

https://hinduismdebunked.com/immorality/cruelty/human/

**Comment by Majestic-Mustang** [comment]
I'm very happy to have found your excellent website!
Can you please add information under 'Bhagavad Gita' so that I could use those as arguments with people who think that it has words from Krishna and everything about it is nice and true.

Thank you so much for this!

**Reply by SageofPHY6Paths** [reply]
Thank you. I didn't get time to update this website in the last few months. But now I got the time and will soon add more verses from different Hindu scriptures.
